src/v/transform/transform_manager.cc

```diff
@@ -342,9 +342,10 @@ ss::future<> manager<ClockType>::handle_plugin_change(model::transform_id id) {
     auto transform = _registry->lookup_by_id(id);
     // If there is no transform we're good to go, everything is shutdown if
     // needed.
-    if (!transform) {
+    if (!transform || transform->paused) {
         co_return;
     }
```

Comment:
Can we update the above comment to explain why we do this and the ramifications (especially on the listing path?)

Reply:
done